CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Catch and report cases where a protected resource is locked out of access.  SDK consumers have a new errs sentinel that allows them to check for this case.
+- Fix a case where missing item LastModifiedTimes could put incremental backups in a permanently failing state.
```

Comment:
nit: wording, it's not permanently breaking the repo, we have a way out of it. Could you please refine this to make it sound not as dire